manual/parts/vspace.tex

```diff
@@ -316,14 +314,14 @@ \section{Mapping Attributes}
 
 \section{Sharing Memory}
 
-seL4 does not allow \obj{Page Table}s to be shared, but does allow
-pages to be shared between address spaces.
-To share a page, the capability to the
-\obj{Page} must first be
-duplicated using the \apifunc{seL4\_CNode\_Copy}{cnode_copy} method and the new copy must
-be used in the \apifunc{seL4\_ARM\_Page\_Map}{arm_page_map} \ifxeightsix or \apifunc{seL4\_x86\_Page\_Map}{x86_page_map} \fi method that maps the page into the second
-address space. Attempting to map the same capability
-twice will result in an error.
+The seL4 kernel does not allow intermediate paging structures (e.g.\ \obj{PageTable} objects) to be
+shared, but it does allow pages to be shared between VSpaces, and VSpaces to be shared by threads.
+
+To share a page, the capability to the \obj{Page} must first be duplicated using the
+\apifunc{seL4\_CNode\_Copy}{cnode_copy} method and the copy must be used in the Map invocation
+(e.g.\ \apifunc{seL4\_ARM\_Page\_Map}{arm_page_map} \ifxeightsix or
+\apifunc{seL4\_x86\_Page\_Map}{x86_page_map}\fi) that maps the page into the second address space.
+Attempting to map the same capability twice will result in an error.
```

Comment:
unless it's overwriting it's previous paging slot which is how page mappings are updated I think

Reply:
Have addressed that now by specifying the target (different tables or vspaces).

We should probably add a section/paragraph about remap somewhere, but I'll leave that for a separate PR.